api/repo.go

```diff
@@ -180,43 +184,36 @@ func CreateRepo(c *gin.Context) {
 		return
 	}
 
-	// TODO: build these from the source client
-	if len(input.GetLink()) == 0 {
-		input.SetLink(url)
-	}
-
-	if len(input.GetClone()) == 0 {
-		input.SetClone(fmt.Sprintf("%s.git", url))
-	}
```

Comment:
These properties are are now returned via `GetRepo` as part of the successful response from the source.

Reply:
This is reporting as "Outdated" because the location moved due to changes happening via another merged change. The comment is still valid, however.